File: SofiForce.Models/Models/EntityModels/RequestStatusModel.cs
using System;

namespace Models
{
    public class RequestStatusModel
    {
        public int RequestStatusId { get; set; }
        public string RequestStatusCode { get; set; }
        public string RequestStatusNameEn { get; set; }
        public string RequestStatusNameAr { get; set; }
        public string Icon { get; set; }
        public string Color { get; set; }
        public int? DisplayOrder { get; set; }
        public bool? IsActive { get; set; }
        public bool? CanEdit { get; set; }
        public bool? CanDelete { get; set; }
        public int? CBy { get; set; }
        public DateTime? CDate { get; set; }
        public int? EBy { get; set; }
        public DateTime? EDate { get; set; }
    }
}

File: SofiForce.Web/ClientApp/src/app/core/Models/EntityModels/RequestStatusModel.ts
export interface RequestStatusModel {
    requestStatusId: number;
    requestStatusCode: string;
    requestStatusNameEn: string;
    requestStatusNameAr: string;
    icon: string;
    color: string;
    displayOrder: number | null;
    isActive: boolean | null;
    canEdit: boolean | null;
    canDelete: boolean | null;
    cBy: number | null;
    cDate: Date | null;
    eBy: number | null;
    eDate: Date | null;
}